modules/serverless-kafka-tls.adoc

```diff
@@ -0,0 +1,80 @@
+// Module is included in the following assemblies:
+//
+// serverless/serverless-kafka.adoc
+
+[id="serverless-configuring-tls-authentication-against-an-apache-kafka_{context}"]
+== Configuring TLS authentication
+
+.Prerequisites
+
+* A Kafka cluster CA certificate as a `.pem` file.
+* A Kafka cluster client certificate and key as `.pem` files.
+
+.Procedure
+
+. Create the certificate files as secrets in your chosen namespace:
++
+----
+$ kubectl create secret --namespace <namespace> generic <kafka-auth-secret> \
+  --from-file=ca.crt=caroot.pem \
+  --from-file=user.crt=certificate.pem \
+  --from-file=user.key=key.pem
+----
++
+[IMPORTANT]
+====
+Use the key names `ca.crt`, `user.crt`, and `user.key`. Do not change them.
+====
+
+
+. Start editing the `KnativeKafka` CR:
++
+----
+$ oc edit knativekafka
+----
+
+. Reference your secret and the namespace of the secret:
++
+----
+apiVersion: operator.serverless.openshift.io/v1alpha1
+kind: KnativeKafka
+metadata:
+  namespace: knative-eventing
+  name: knative-kafka
+spec:
+  channel:
+    authSecretName: <kafka-auth-secret>
+    authSecretNamespace: <kafka-auth-secret-namespace>
+    bootstrapServers: <bootstrap-server>
+    enabled: true
+  source:
+    enabled: true
+----
++
+[NOTE]
+====
+Make sure to specify the matching port in the bootstrap server.
+====
++
+For example:
++
+----
+apiVersion: operator.serverless.openshift.io/v1alpha1
+kind: KnativeKafka
+metadata:
+  namespace: knative-eventing
+  name: knative-kafka
+spec:
+  channel:
+    authSecretName: tls-user
+    authSecretNamespace: kafka
+    bootstrapServers: eventing-kafka-bootstrap.kafka.svc:9093
```

Comment:
```suggestion
    bootstrapServers: eventing-kafka-bootstrap.kafka.svc:9094
```

Let's use 9094 here since that's the traditional TLS port for Kafka.